### DISCUSSION TITLE
Unable to access/show XAML designer

### DISCUSSION THREAD

**Discussion by olivegamestudio** [unoplatform/uno#9141] [Q&A]
### Current behavior

From Visual Studio 2022 Preview, no XAML designer is shown - even selecting Windows from the top left hand corner and closing and reopening just shows the XAML only.

### Expected behavior

The XAML designer would be nice to see along side the XAML itself.

**Comment by jeromelaban** [comment]
Thanks for the feedback. You'll notice that when you create a "vanilla" WinAppSDK application, there's no designer either, as Microsoft has [not yet put a priority on this feature](https://github.com/microsoft/microsoft-ui-xaml/issues/5917#issuecomment-1028626378).
